cmd/notary-server/main.go

```diff
@@ -32,6 +34,10 @@ import (
 const (
 	jsonLogFormat = "json"
 	DebugAddress  = "localhost:8080"
+	// This is the generally recommended maximum age for Cache-Control headers
+	// (one year, in seconds, since one year is forever in terms of internet
+	// content)
+	maxMaxAge = 31536000
```

Comment:
After the discussions about the expiries would `60 * 60 * 24 * 365` be clearer about what the value is (without reading the comment)?